app/services/reports/children_served_report_service.rb

```diff
@@ -39,12 +34,53 @@ def average_children_monthly
 
     # @return [Float]
     def per_child_monthly
+      total_distributions = organization.distributions.for_year(year).count
+      total_kits = organization.kits.count
+
+      total_avg = if total_distributions.zero? && total_kits.zero?
+        0.0
+      else
+        (loose_disposable_distribution_average * total_distributions + kit_average * total_kits) / (total_distributions + total_kits)
+      end
+
+      total_avg.nan? ? 0.0 : total_avg
+    end
+
+    private
+
+    def loose_disposable_distribution_average
+      organization
+      .distributions
+      .for_year(year)
+      .joins(line_items: :item)
+      .merge(Item.disposable)
+      .average('COALESCE(items.distribution_quantity, 50)') || 0.0
+    end
+
+    def kit_average
+      organization
+      .kits
+      .joins(inventory_items: :item)
+      .merge(Item.disposable)
+      .average('COALESCE(inventory_items.quantity, 0)') || 0.0
+    end
+
+    def distributed_loose_disposable_items
+      organization
+      .distributions
+      .for_year(year)
+      .joins(line_items: :item)
+      .merge(Item.disposable)
+      .sum('line_items.quantity / COALESCE(items.distribution_quantity, 50)')
+    end
+
+    def distributed_kits_with_disposable_items
       organization
-        .distributions
-        .for_year(year)
-        .joins(line_items: :item)
-        .merge(Item.disposable)
-        .average('COALESCE(items.distribution_quantity, 50)') || 0.0
+      .kits
+      .joins(line_items: :item)
+      .merge(Item.disposable)
+      .distinct
+      .count("kits.id")
     end
```

Comment:
If I read this right,  this is giving the number of kits defined with disposables in them, rather than the number of kits distributed with disposables.

Reply:
Ahhh you are right, I'm working on fixing this. Having a bit of trouble returning only distributed kits with disposables

Reply:
@cielf I think I finally got this working as intended. Just pushed up